vulture_os/toolkit/api_parser/reachfive/reachfive.py

```diff
@@ -140,9 +161,11 @@ def execute(self):
         cpt = 0
         page = 1
         total = 1
-        last_datetime = self.frontend.last_api_call.isoformat(timespec="milliseconds").replace("+00:00", "Z")
+        since = self.frontend.last_api_call.isoformat(timespec="milliseconds")
+
         while cpt < total:
-            status, tmp_logs = self.get_logs(page=page, since=self.last_api_call)
+
```

Comment:
Try to always allow the parser to quit when asked
```suggestion
        while cpt < total and not self.evt_stop.is_set():
```